common/xbps-src/shutils/lint_commits.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+lint_commits() {
+	local base="$1"
+	local status=0
+	local cmt
+	if [ -n "$base" ]; then
+		base=$("$XBPS_GIT_CMD" rev-parse --verify "$base") ||
+			return
+	elif base=$("$XBPS_GIT_CMD" rev-parse --verify FETCH_HEAD 2>/dev/null); then
+		:
+	elif base=$("$XBPS_GIT_CMD" rev-parse --verify ORIG_HEAD 2>/dev/null); then
+		:
+	else
+		echo base commit not found >&2
+		exit 1
+	fi
+
+	for cmt in $("$XBPS_GIT_CMD" rev-list --reverse --abbrev-commit "$base"..HEAD)
```

Comment:
I think it could take a second optional argument to override `HEAD` here, what do you think? Otherwise it will be harder to run locally.

Reply:
Implemented.